perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/RegistrarManager.java

```diff
@@ -274,7 +275,23 @@
 	 * @return list of applications
 	 * @throws PerunException
 	 */
-	List<Application> getApplicationsForVo(PerunSession sess, Vo vo, List<String> state) throws PerunException;
+	List<Application> getApplicationsForVo(PerunSession sess, Vo vo, List<String> state, Boolean includeGroupApplications) throws PerunException;
+
+
+	/**
+	 * Gets all applications in a given state for a given VO.
+	 * If state is null, returns all applications for a given VO.
+	 *
+	 * @param sess who is asking
+	 * @param vo VO to get applications for
+	 * @param state application state to filter by
+	 * @param dateFrom return only applications with this date or newer
+	 * @param dateTo return only applications with this date or older
+	 * @return list of applications
```

Comment:
Please add ``includeGroupApplications`` param to the javadoc. Same for the method above.

Reply:
Fixed.